src/Validator/ValidatorInterface.php

```diff
@@ -21,7 +21,7 @@
      *
      * @return bool
      */
-    public function isValid();
+    public function __invoke();
```

Comment:
Well, I'll be. Right you are. Must've been something else in there breaking all the tests for validators then.